dpnp/backend/custom_kernels_elemwise.cpp

```diff
@@ -46,7 +46,7 @@
             size_t i = global_id[0]; /*for (size_t i = 0; i < size; ++i)*/                                             \
             {                                                                                                          \
                 _DataType_output input_elem = array1[i];                                                               \
-                result[i] = __operation__;                                                                             \
+                result[i] = __operation__(input_elem);                                                                 \
```

Comment:
please keep `input_elem` as before. I would propose to keep this flexibility for this moment.